.github/workflows/main.yml

```
@@ -21,11 +21,11 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [16.x]
+        node-version: [16.x, 18.x]
```

Karma is doing the testing, so just need to run it once with one node version.
```suggestion
        node-version: [18.x]
```